tests/acceptance/async_adapter_tests.py

```diff
@@ -33,8 +33,11 @@ class TestConfirmSelect(AsyncTestCase, AsyncAdapters):
     DESCRIPTION = "Receive confirmation of Confirm.Select"
 
     def begin(self, channel):
-        channel._on_selectok = self.on_complete
-        channel.confirm_delivery()
+        channel.confirm_delivery(ack_nack_callback=self.ack_nack_callback,
+                                 callback=self.on_complete)
+
+    def ack_nack_callback(frame):
+        pass
```

Comment:
Did you intend this to do something inside ack_nack_callback?

Reply:
Nope, we're requiring a callable for the `ack_nack_callback` parameter so I did it this way.

Reply:
got it, thanks!